Makefile

```diff
@@ -202,6 +202,16 @@ format:
 	@./tools/format_all.sh
 	@make --no-print-directory -C test format
 
+.PHONY: typos
+typos:
+	@cargo install typos-cli
```

Comment:
This new tool should be installed in our Docker image. To do so, update the Dockerfile image and bump the version file using `tools/bump_version.sh` in a separate PR. After merging this PR, the new Docker image will be generated and uploaded to Docker Hub automatically.

Reply:
If you have more questions regarding Docker image creation, ask @StevenJiang1110 or @junyang-zh.